.env.example

```diff
@@ -2,4 +2,5 @@ VITE_PROJECT_ACCESS_KEY=AQAAAAAAADVH8R2AGuQhwQ1y8NaEf1T7PJM
 VITE_WAAS_CONFIG_KEY="eyJwcm9qZWN0SWQiOjEzNjM5LCJycGNTZXJ2ZXIiOiJodHRwczovL3dhYXMuc2VxdWVuY2UuYXBwIn0="
 VITE_GOOGLE_CLIENT_ID=970987756660-35a6tc48hvi8cev9cnknp0iugv9poa23.apps.googleusercontent.com
 VITE_APPLE_CLIENT_ID=com.horizon.sequence.waas
-VITE_WALLET_CONNECT_ID='c65a6cb1aa83c4e24500130f23a437d8'
\ No newline at end of file
+VITE_WALLET_CONNECT_ID='c65a6cb1aa83c4e24500130f23a437d8'
+VITE_PROJECT_ID=
```

Comment:
We should provide a project ID by default (Dim Sum Buddy)

Reply:
I deleted that .env.example example. It isn't necessary.

Reply:
Done